extensions/wikia/AdEngine/js/recovery/message.js

```diff
@@ -42,13 +44,26 @@ define('ext.wikia.adEngine.recovery.message', [
 		logGroup = 'ext.wikia.adEngine.recovery.message',
 		localStorageKey = 'rejectedRecoveredMessage';
 
+	function track(action) {
+		tracker.track({
+			'ga_category': 'ads-recovery-message',
```

Comment:
I'm afraid we have to change the dictionary here. If nothing changes (@KrystianMistrzak could tell us about the changes, I guess) [we are suppose to use](https://github.com/Wikia/app/blob/dev/resources/wikia/modules/tracker.js#L209) one of the predefined `Wikia.Tracker.ACTIONS`. Do we have category `ads` in our regular GA already? If not I'd go with:
```
{
  'ga': 'ads',
  'ga_action': Wikia.Tracker.ACTIONS.CLICK, // or Wikia.Tracker.ACTIONS.IMPRESSION
  'ga_label': 'ads-recovery-message',
  'ga_value': 0,
  trackingMethod: 'analytics'
}
```
What do you think?